### PR TITLE
Make ContractFunctionResult.get* methods default to 0

### PR DESCRIPTION
**Pull request by Sheng-Long**

### CODE REVIEW

File: src/contract/ContractFunctionResult.ts [hunk 1]
```diff
@@ -65,7 +65,11 @@ export class ContractFunctionResult {
         return this.bytes;
     }
 
-    public getString(index: number): string {
+    public getString(index?: number): string {
+        if (index == null) {
+            return Buffer.from(this.getBytes(0)).toString("utf-8");
+        }
+
         return Buffer.from(this.getBytes(index)).toString("utf-8");
```
Comment (mehcode) [inline src/contract/ContractFunctionResult.ts]: Do `index ?? 0` instead here

File: src/contract/ContractFunctionResult.ts [hunk 2]
```diff
@@ -65,7 +65,11 @@ export class ContractFunctionResult {
         return this.bytes;
     }
 
-    public getString(index: number): string {
+    public getString(index?: number): string {
```
Comment (mehcode) [inline src/contract/ContractFunctionResult.ts]: We should make this change for all accessors

Comment (Sheng-Long) [reply src/contract/ContractFunctionResult.ts]: Should this be spun out as an issue, or should it be included in this pr?

Comment (mehcode) [reply src/contract/ContractFunctionResult.ts]: this PR, we shouldn't merge in a one-off change like this

**Pull request merged**